ecoscope/io/earthranger.py

```diff
@@ -708,6 +579,28 @@ def get_events(
         df.sort_values("time", inplace=True)
         return df
 
+    def get_patrols(self, filter=None, status=None, **addl_kwargs):
+        """
+        Parameters
+        ----------
+        filter:
+            example: {"date_range":{"lower":"2020-09-16T00:00:00.000Z"}}
```

Comment:
Think we should provide since and until parameters here rather than a single date range parameter which is harder for the user to construct.